lib/std/macho.zig

```diff
@@ -24,6 +24,20 @@ pub const mach_header_64 = extern struct {
     reserved: u32,
 };
 
+pub const fat_header = extern struct {
+    magic: u32,
+    nfat_arch: u32,
+};
+
+pub const fat_arch = extern struct
```

Comment:
The formatting looks wonky, please run `zig fmt` over all the files you touch (that's a pre-requisite for the CI to pass).